numpy/distutils/fcompiler/environment.py

```diff
@@ -33,7 +33,8 @@ def __getattr__(self, name):
         try:
             conf_desc = self._conf_keys[name]
         except KeyError:
-            raise AttributeError(name)
+            raise AttributeError("'dict' object has no attribute %s" % name) from None
```

Comment:
```suggestion
            raise AttributeError("'EnvironmentConfig' object has no attribute '%s'" % name) from None
```

Reply:
Great. Was curious about exposing class variable which you encoded with 'EnvironmentConfig' 👍